# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/nvme-cli/hotfix-setlocale.patch.musl
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- nvme-cli-2.11/nvme-print.c.vanilla	2024-12-06 15:24:38.053228824 +0100
+++ nvme-cli-2.11/nvme-print.c	2024-12-06 15:27:08.676561164 +0100
@@ -801,11 +801,12 @@
 {
 	const char *locale, *underscore;
 	char country[3] = { 0 };
-
+#if 0
 	setlocale(LC_MEASUREMENT, "");
 	locale = setlocale(LC_MEASUREMENT, NULL);
 
 	if (!locale || strlen(locale) < 2)
+#endif
 		return false;
 
 	underscore = strchr(locale, '_');
